templates/terraform/resource.erb

```diff
@@ -552,7 +552,9 @@ func resource<%= resource_name -%>Import(d *schema.ResourceData, meta interface{
 <%= lines(compile(object.custom_code.custom_import)) -%>
 <% else -%>
     config := meta.(*Config)
-    if err := parseImportId([]string{"<%= import_id_formats(object).map{|s| format2regex s}.join('","') -%>"}, d, config); err != nil {
+    if err := parseImportId([]string{
+        "<%= import_id_formats(object).map{|s| format2regex s}.join("\",\n\"") -%>",
```

Comment:
I think I'd prefer a for loop (wrote it down to see if I like it more) rather than joining with newline that is actually meant to be output into the template, but it's not a big deal especially if whitespace get weird.  

```
<% for id in import_id_formats(object) -%>
        "<%= id.map{|s| format2regex s} %>",
<% end -%>
```